runtime/heiko/src/lib.rs

```diff
@@ -432,8 +432,7 @@ parameter_types! {
     pub const StakingPalletId: PalletId = PalletId(*b"par/lqsk");
     pub const StakingCurrency: CurrencyId = CurrencyId::KSM;
     pub const LiquidCurrency: CurrencyId = CurrencyId::xKSM;
-    pub const MaxWithdrawAmount: Balance = 1_000_000_000_000_000;
-    pub const MaxAccountProcessingUnstake: u32 = 5;
+    pub Agent: MultiLocation = X2(Parent, Parachain(ParachainInfo::parachain_id().into()));
```

Comment:
if we use parachain's holding account, then we will need xcm::Transact to be able to control funds on relaychain